boards/sim/sim/sim/configs/ostest/defconfig

```diff
@@ -10,10 +10,12 @@ CONFIG_ARCH_BOARD="sim"
 CONFIG_ARCH_BOARD_SIM=y
```

Comment:
This change seems out of place with the other commits in the PR.  Perhaps it should be a separate PR?

Reply:
this change is required by other changes as otherwise ./nuttx just hangs after running the tests.